site/data/docs/custom-authentication.mdx

```diff
@@ -95,4 +149,6 @@ export default function App({ Component, pageProps }: AppProps) {
 }
 ```
 
-If you've got this far and created an adapter for an existing open source authentication library, please consider creating a package for others to use!
+Full working example:
```

Comment:
Let's revert this. We'd like to feature plug-and-play adapters rather than a working example; SIWE is the working example.

Reply:
SIWE isn't using Rainbowkit in their example, in your example you're depending on next-auth, this does totally different thing.

Reply:
@smakosh The goal for this section is to add adapters based on the Custom Authentication adapter spec for libraries like Auth.js